realtime/utils.js

```diff
@@ -195,9 +198,16 @@ function perspectiveEmit(nspComponents, obj) {
    * If the obj is a subject, just apply the subjectTagFilter and return the
    * result.
    */
-  if (isThisSubject(obj)) return applyFilter(subjectTagFilter, obj.tags);
+  const objectType = whatAmI(obj);
+  if (objectType === ObjectType.Subject) {
+    return applyFilter(subjectTagFilter, obj.tags);
+  }
+
+  if (objectType === ObjectType.Aspect) {
+    return applyFilter(aspectFilter, obj.aspect.name) &&
+      applyFilter(aspectTagFilter, obj.tags);
+  }
```

Comment:
Can you also add these changes to the realtime app?

Reply:
https://github.com/salesforce/refocus-real-time/pull/8